backend/src/create.rs

```diff
@@ -481,3 +496,23 @@ fn nodes_to_edges(graph: &Graph, nodes: Vec<utils::osm2graph::IntersectionID>) -
     }
     edges
 }
+
+fn get_poi(tags: &Tags, point: Coord) -> Option<POI> {
+    if tags.is_any("shop", vec!["convenience", "grocery", "supermarket"]) {
```

Comment:
These seem to cover most, but there might be niche things like fruit stands tagged differently. We can validate with local knowledge and iterate.